docs/templates/installation/deploy_polyaxon.md

```diff
@@ -176,6 +176,15 @@ ingress:
 serviceType: LoadBalancer
 ```
 
+It is however recommended to enable RBAC and start minikube with the option `--extra-config=apiserver.authorization-mode=RBAC`.
+
+After installing polyaxon, use the following command to enable access to the API service:
```

Comment:
This could be an option as well as using the `NodePort` service and port forwarding. Maybe it could be the easiest option for users.

 I will add that after merging these changes